spec/factories_spec.rb

```diff
@@ -3,7 +3,7 @@
 RSpec.describe 'FactoryBot' do
   FactoryBot.factories.each do |factory|
     it "#{factory.name} should pass lint" do
-      FactoryBot::Linter.new([factory]).lint!
+      expect(FactoryBot::Linter.new([factory]).lint!).not_to raise(Error)
```

Comment:
rspec examples need to have an assertion, I figured this would be a good way around this error